pvt_validation.ipynb

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # weird, pvt_andhra_pradesh has unmatching duplicates
</pre><p>Was this the only place where this happened? I wonder if it was an issue with types. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>